webserver/src/main/java/io/kestra/webserver/controllers/api/TemplateController.java

```diff
@@ -88,6 +90,7 @@ public PagedResults<Template> find(
     public HttpResponse<Template> create(
         @Parameter(description = "The template") @Valid @Body Template template
     ) throws ConstraintViolationException {
+        template.setTenantId(MAIN_TENANT);
```

Comment:
Should be (please apply everywhere)
```suggestion
        template.setTenantId(tenantService.resolveTenant());
```